stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
+        length = length + string.len("speed []") + speed:len()
+        filters[#filters+1] = "speed" .. it(" [" .. speed .. "]")
+    end
+
+    if prop == "vf" and mp.get_property_bool("deinterlace") then
```

Comment:
This property being set does _not_ indicate a deinterlacing filter being used, the relation doesn't work like that. It's also a property with 3 states so you can't really evaluate it with a bool. 
Other than that, "deinterlace" is not a filter. E.g. yadif/pulldown/etc are but these get inserted into `vf` when applied and therefore show up there anywah.